test/start-kubernetes.sh

```diff
@@ -55,22 +55,21 @@ SSH_TIMEOUT=60
 SSH_ARGS="-oIdentitiesOnly=yes -oStrictHostKeyChecking=no \
         -oUserKnownHostsFile=/dev/null -oLogLevel=error \
         -i ${SSH_KEY}"
-SSH_ARGS+=" -oServerAliveInterval=1" # required for disruptive testing, to detect quickly when the machine died
+SSH_ARGS+=" -oServerAliveInterval=15" # required for disruptive testing, to detect quickly when the machine died
```

Comment:
why separate += for adding ServerAliveInterval? For comment? We can still keep comment if all options are assigned at once

Reply:
> For comment?

Yes.

> We can still keep comment if all options are assigned at once

It's harder to correlate the comment with the option when doing that - one basically has to repeat the "-oServerAliveInterval" in the comment.